src/_guides/language/language-tour.md

```diff
@@ -2891,32 +2921,89 @@ class SmartTelevision extends Television {
 }
 {% endprettify %}
 
-Subclasses can override instance methods, getters, and setters. Here’s
-an example of overriding the Object class’s `noSuchMethod()` method,
-which is called whenever code attempts to use a non-existent method or
-instance variable:
 
-<!-- language-tour/no-such-method/bin/main.dart -->
+#### Overriding members
+
+Subclasses can override instance methods, getters, and setters.
+You can use the `@override` annotation to indicate that you are
+intentionally overriding a member:
+
+<!-- language-tour/overrides/bin/override.dart -->
 {% prettify dart %}
-class A {
-  // Unless you override noSuchMethod, using a
-  // non-existent member results in a NoSuchMethodError.
-  void noSuchMethod(Invocation mirror) {
-    print('You tried to use a non-existent member:' +
-          '${mirror.memberName}');
+class SmartTelevision extends Television {
+  @override
+  void turnOn() {
+    // ...
   }
+  // ...
 }
 {% endprettify %}
 
-You can use the `@override` annotation to indicate that you are
-intentionally overriding a member:
+To narrow the type of a method parameter while maintaining type safety,
+use the `covariant` keyword.
```

Comment:
I've drastically reduced the coverage of covariant. Now we just point to the Sound Dart docs.

I've removed the examples, so I'll close the PR that added them.